src/cloud-cli/cli.ts

```diff
@@ -70,6 +71,39 @@ program
     await getAppLogs(options.name, options.host, options.port);
   });
 
+const userdb = program
+  .command('userdb');
+   
+
+userdb
+  .command('create')
+  .argument('<string>', 'database name')
+  .option('-h, --host <string>', 'Specify the host', 'localhost')
+  .option('-p, --port <port>', 'Specify the port', '8080')
+  .option('-a, --admin <admin>', 'Specify the admin user', 'postgres')
+  .option('-W, --password <admin>', 'Specify the admin password', 'postgres')
+  .option('-s, --sync', 'make synchronous call', false)
+  .action((async (dbname: string, options: { host: string, port: string, admin: string, password: string, sync: boolean }) => {
+    await createUserDb(options.host, options.port, dbname, options.admin, options.password, options.sync)
+  }))
+
+userdb
+  .command('status')
+  .argument('<string>', 'database name')
+  .option('-h, --host <string>', 'Specify the host', 'localhost')
+  .option('-p, --port <port>', 'Specify the port', '8080')
+  .action((async (dbname: string, options: { host: string, port: string }) => {
+    await getUserDb(options.host, options.port, dbname)
+  })) 
+
+userdb
+  .command('delete')
+  .argument('<string>', 'database name')
+  .option('-h, --host <string>', 'Specify the host', 'localhost')
+  .option('-p, --port <port>', 'Specify the port', '8080')
```

Comment:
Is there a way to make the `-h` and `-p` arguments required by the parent command `userdb` so they apply to all the sub commands?

Reply:
Tried it . did not work